src/Stamp/MessageId.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Messages\Stamp;
+
+use Ramsey\Uuid\Uuid;
+use Ramsey\Uuid\UuidInterface;
+use Symfony\Component\Messenger\Stamp\StampInterface;
+
+readonly class MessageId implements StampInterface
+{
+    private UuidInterface $messageId;
+
+    public function __construct()
+    {
+        $this->messageId = Uuid::uuid4();
+    }
+
+    public function getMessageId(): string
+    {
+        return $this->messageId->toString();
+    }
```

Comment:
may as well use the more concise form since we're in PHP 8.3 now?

```suggestion
    public UuidInterface $messageId;

    public function __construct()
    {
        $this->messageId = Uuid::uuid4();
    }
```

Reply:
actually realised my alternative isn't exactly the same, since I lost the toString. But I don't think there's anything calling it right now, and if there is `$stamp->messageId->toString() ` doesn't seem much worse than `$stamp->getMessageId()`, and allows a bit more flexibility.

Reply:
We do use the concise string getter for the current MatchBot copy of this class, and don't yet have a use case for the value object more generally, so I'm not sure I see the advantage of the change clearly.

Reply:
OK, not a blocker. I mostly just think the change is making the class shorter and simpler to read.